pkgs/by-name/si/silx/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "silx";
+  version = "2.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-LfCRWkUrqQb7zxiFTPhy/g9FWhNMXTRbhEgek4tZb5I=";
+  };
+
+  nativeBuildInputs = [ python3Packages.cython ];
+
+  propagatedBuildInputs = with python3Packages; [
+    h5py
+    numpy
+    matplotlib
+    pyopengl
+    python-dateutil
+    pyside6
+    fabio
+  ];
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Reply:
Fixed this (twice) by making `lib.maintainers` and `lib.licenses` explicit. Hope that was the original issue.